sjsonnet.Error: Internal Error
    at [std.nativePanic].(native_panic.jsonnet:1:26)
Caused by: java.lang.RuntimeException: native function panic
